docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,414 @@
+# vmanomaly Quickstart
+
+**Prerequisites**
+- In the tutorial, we'll be using the following VictoriaMetrics components:
+  -  [VictoriaMetrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html) (v.1.83.1)
+  -  [vmalert](https://docs.victoriametrics.com/vmalert.html) (v.1.83.1)
+  -  [vmagent](https://docs.victoriametrics.com/vmagent.html) (v.1.83.1)
+  
+  If you're unfamiliar with the listed components, please read [QuickStart](https://docs.victoriametrics.com/Quick-Start.html) first.
+- It is assumed that you are familiar with [Grafana](https://grafana.com/)(v.9.3.1) and [Docker](https://docs.docker.com/get-docker/).
```

Comment:
- It is assumed that you are familiar with [Grafana](https://grafana.com/oss/) (v.9.3.1) , [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/)